app/controllers/accounts_controller.rb

```diff
@@ -63,7 +58,7 @@ def update
   # DELETE /accounts/1 or /accounts/1.json
   def destroy
     # @account.destroy!
-    @account.update(:status_id => 3)
```

Comment:
Was this pulled from main before merging new changes because delete was updated? Can you check with Ryan O, and confirm if inactive user is still ID#3, because as mentioned in below, it should be #4. After this, it can be merged.
https://robertyam.atlassian.net/browse/ROT-39

Account status changed: 1: admin, 2: officer, 3: member, 4: inactive.